.bundlemonrc

```diff
@@ -0,0 +1,78 @@
+{
+    "baseDir": "./build",
+    "files": [
+        {
+            "path": "1/mespapiers.<hash>.js"
+        },
+        {
+            "path": "2/mespapiers.<hash>.js"
+        },
+        {
+            "path": "3/mespapiers.<hash>.js"
+        },
+        {
+            "path": "4/mespapiers.<hash>.js"
+        },
+        {
+            "path": "5/mespapiers.<hash>.js"
+        },
+        {
+            "path": "6/mespapiers.<hash>.js"
+        },
+        {
+            "path": "7/mespapiers.<hash>.js"
+        },
+        {
+            "path": "8/mespapiers.<hash>.js"
+        },
+        {
+            "path": "9/mespapiers.<hash>.js"
+        },
+        {
+            "path": "10/mespapiers.<hash>.js"
+        },
+        {
+            "path": "11/mespapiers.<hash>.js"
+        },
+        {
+            "path": "12/mespapiers.<hash>.js"
+        },
+        {
+            "path": "13/mespapiers.<hash>.js"
+        },
+        {
+            "path": "14/mespapiers.<hash>.js"
+        },
+        {
+            "path": "15/mespapiers.<hash>.js"
+        },
+        {
+            "path": "16/mespapiers.<hash>.js"
+        },
+        {
+            "path": "17/mespapiers.<hash>.js"
+        },
+        {
+            "path": "18/mespapiers.<hash>.js"
+        },
+        {
+            "path": "19/mespapiers.<hash>.js"
+        },
+        {
+            "path": "app/mespapiers.<hash>.js"
+        },
+        {
+            "path": "app-mespapiers.<hash>.min.css"
+        },
+        {
+            "path": "vendors/mespapiers.<hash>.js"
+        },
+        {
+            "path": "vendors-mespapiers.<hash>.<hash>.min.css"
+        },
+        {
+            "path": "img/icon.<hash>.svg"
+        }
+    ],
+    "reportOutput": ["github"]
```

Comment:
nit: I recommend to use eitheer to maxIncrement or the maxSize to limit big changes that reviewers didn't detect while reading bundlemon reports

Reply:
maybe we should do it after: https://github.com/cozy/mespapiers/pull/183